src/components/ProfileCard/ProfileCard.stories.tsx

```diff
@@ -0,0 +1,374 @@
+import { Box } from '@mui/material';
+
+import { ProfileCard } from './ProfileCard';
+import { PROFILE_CARD_SIZES, PROFILE_CARD_VARIANTS } from './ProfileCard.constants';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Components/ProfileCard',
+  component: ProfileCard,
+  parameters: {
+    layout: 'centered',
+    docs: {
+      description: {
+        component:
+          'A reusable profile card component for displaying user information and statistics.',
+      },
+    },
+  },
+  argTypes: {
+    name: {
+      control: 'text',
+      description: 'User full name',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    username: {
+      control: 'text',
+      description: 'User username/handle',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    avatarUrl: {
+      control: 'text',
+      description: 'URL for the user avatar image',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'undefined' },
+      },
+    },
+    role: {
+      control: 'text',
+      description: 'User role or title',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'undefined' },
+      },
+    },
+    size: {
+      control: 'select',
+      options: Object.keys(PROFILE_CARD_SIZES),
```

Comment:
Using `Object.keys(PROFILE_CARD_SIZES)` returns string keys, which may be acceptable but is less precise than `Object.values(...)`. If the mapping is `Record<ProfileCardSize, ProfileCardSize>` you likely want the actual values, not keys cast as strings.

**Sugerencia:**
```typescript
Use `Object.values(PROFILE_CARD_SIZES)` to provide the selectable values:

options: Object.values(PROFILE_CARD_SIZES),

This ensures storybook controls present the actual allowed values and not intermediate keys.
```